### PR TITLE
Update SingleFileApiTests

### PR DESCRIPTION
**Pull request by mateoatr**

### CODE REVIEW

**Comment by mateoatr** [comment]
PTAL @agocke

**Comment by agocke** [comment]
Is this just the cleanup from earlier where we can share the code between the existing tests instead of having new tests?

**Comment by mateoatr** [comment]
That's correct.

File: src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/SingleFileApiTests.cs [hunk 1]
```diff
@@ -37,20 +40,25 @@ public void SelfContained_SingleFile_APITests()
                 // For single-file, Environment.GetCommandLineArgs[0] should return the file path of the host.
                 .And.HaveStdOutContaining("Command line args: " + singleFile)
                 .And.HaveStdOutContaining("ExecutingAssembly.Location: " + Environment.NewLine)
-                .And.HaveStdOutContaining("AppContext.BaseDirectory: " + Path.GetDirectoryName(singleFile));
+                .And.HaveStdOutContaining("AppContext.BaseDirectory: " + Path.GetDirectoryName(singleFile))
+                // If we don't extract anything to disk, the extraction dir shouldn't
+                // appear in the native search dirs.
+                .And.HaveStdOutMatching($"NATIVE_DLL_SEARCH_DIRECTORIES: .*{Regex.Escape(bundleDir)}")
```
Comment (agocke) [inline src/installer/tests/Microsoft.NET.HostModel.Tests/AppHost.Bundle.Tests/SingleFileApiTests.cs:46]: Why not verify the list exactly? The variable shouldn't change much, should it?

**Pull request closed without merging**